docs/schema/fulltextindex-element-managedpropertyinfo-complextypesps15xsdsearchset2.md

```diff
@@ -16,7 +16,7 @@ description: "Last modified: March 09, 2015"
   
 ## Element information
 
-|||
+|Information|Location|
```

Comment:
Sue is using HTML encoding for a non-breaking space.  I'm unsure which is a better solution, but I'll just give you the idea for now.  Implement if it makes sense, keep your current standard if it doesn't.  For example:

| &nbsp | &nbsp |
|:-----|:-----|
|**Element type** <br/> |xs:string  <br/> |
|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration  <br/> |